index.html

```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(3).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    
+        <section class="box1">
+            <img src="resources/Rectangle 21(4).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(5).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    </section>
+
+    <section>
+        <h1>About</h1>
```

Comment:
Kindly, by the accessibility rule there can only be one H1 tag on your page, the others follow in order of importance, H2, H3, etc.

Reply:
@mgmediaweb I will make the changes, no prob. But are you sure about this? Could you pass some guidelines that point this out? I've looked this up and I find that you can use more than one h1, just not nested within themselves. I don't mind complying, but I would love to understand the matter a little bit better. Please send guidelines that defend the case. Thanks.

Reply:
@mgmediaweb, for SEO reasons you should always use only one `<h1>` tag on the page, that represent a title or the most important header in that page